<launch>

	<node pkg="bitathome_hardware_control" name="hc_cmd_interface" type="hc_cmd_interface.py" />

	<node pkg="bitathome_move_control" name="hc_avoid_obstacle" type="hc_avoid_obstacle.py" output="screen" />

</launch>
